<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface_main</class>
 <widget class="QMainWindow" name="interface_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helcza Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/icon_resource.qrc">
    <normaloff>:/icon/icon/icon2.bmp</normaloff>:/icon/icon/icon2.bmp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{border-image: url(:/image/image/fon.jpg);}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button_open_camera_menu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string>Open camera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_open_pattern_menu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string>Pattern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_open_image_menu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string>Open image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(90, 131, 151);</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(90, 131, 151);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource/icon_resource.qrc"/>
  <include location="resource/image_resource.qrc"/>
 </resources>
 <connections/>
</ui>
